### ISSUE TITLE
Update Apache SSL Documentation Re: Error: Files must be fully qualified with custom_fragment

### ISSUE THREAD

**Issue by ReservedDeveloper** [puphpet/puphpet#1807]
Realize this has been reported previously in #1717 and #1757, I'm not intending to ruffle any feathers. However, I did want to point out that some of the comments left there don't seem to jive with current documentation.

As of this ticket's creation, the documentation on [PuPHPet](http://puphpet.com/) states the following when checking the 'Enable SSL' flag on Web Servers-->Apache-->Virtual Host:

`Add Redirect permanent / https://{YOUR_VHOST_NAME}/ to the non-SSL vhost's Custom Directives to redirect all traffic to the SSL vhost.`

Removing the offending file() wrap as pulled in on [8525b77](https://github.com/puphpet/puphpet/commit/8525b77986f7b971c4992b249b2706cc4f0e8fde#diff-173e3be166cd3f7b392bdf365b427b2dR160) seems to resolve the issue, and I'm able to run vagrant up/provision without issue.

That said, I'm not sure of the implications of such an approach, so I'm not certain whether the file() check or an update to the documentation would be more appropriate.

**Comment by jtreminio** [comment]
Custom Directives must be defined as a path to a file that contains your directives.

I understand your point, though. How would you suggest I reword this to make it crystal clear that Custom Directives shouldn't contain actual code, but a file path?

**Comment by ReservedDeveloper** [comment]
Truth be told, this may really boil down to not being able to guard against operator error :laughing:. 

Not sure how much clearer you can be between the `/path/to/file` placeholder in the custom directive config field and mentioning of: `File must exist inside VM on initial $ vagrant up!` in the tooltip.

**Comment by jeeftor** [comment]
I've also managed to get confused on this one here... as to where i should stick this file :)
